frame/bags-list/src/list/mod.rs

```diff
@@ -643,8 +648,13 @@ impl<T: Config> Node<T> {
 	}
 
 	/// Get the underlying voter.
-	// TODO: this is public for remote-ext test
-	pub fn id(&self) -> &T::AccountId {
+	pub(crate) fn id(&self) -> &T::AccountId {
+		&self.id
+	}
+
+	/// Get the underlying voter (publicly accessible for testing and benchmarks).
+	#[cfg(feature = "std")]
```

Comment:
I think benchmark run with the wasm runtime, so without std. You probably want to add `any(.., feature = runtime-benchmarks)`

Reply:
my previous comment was misleading because I copied and pasted that last bit - for now this is mainly used in some integration tests in the polkadot repo